mandalab_visual/mandalab_visual.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+from pyvis.network import Network
+from mandalab_visual.consts import *
+
+DEFAULT_GRAPH_NAME_FORMAT = "{graph_base}_{name}_{timestamp}.html"
+
+
+def guess_if_running_in_notebook():
```

Comment:
consider changing the name to `is_running_in_notebook`
maybe doc the "guess part"